components/browser/engine-system/src/main/res/values-nv/strings.xml

```diff
@@ -2,4 +2,9 @@
 <resources>
     <!-- Text for the title of an alert dialog displayed by a web page. %1$s will be replaced with the URL of the current page (displaying the dialog). -->
     <string name="mozac_browser_engine_system_alert_title">Níłch’i naaltsoos -gi ’ání %1$s:</string>
-    </resources>
+    <!-- Text for the message of an auth dialog displayed by a web page.
+    %1$s will be replaced by the realm, %2$s will be replaced with the URL of the current page (displaying the dialog). -->
+    <string name="mozac_browser_engine_system_auth_message">chooʼį́ ʼázhiʼ choiníłʼínígíí dóó ʼáchʼą́ą́h saad binahjįʼ yah ʼanánídááhígíí niyókeed. Siteígíí ʼání:</string>
```

Comment:
@Pike this seems to be missing parameters?

CI lint is failing with following errors:
```[task 2020-05-06T00:14:56.201Z]   /builds/worker/checkouts/src/components/browser/engine-system/src/main/res/values/strings.xml:10: Error: Inconsistent number of arguments in formatting string mozac_browser_engine_system_auth_message; found both 0 and 2 [StringFormatCount]
[task 2020-05-06T00:14:56.201Z]       <string name="mozac_browser_engine_system_auth_message">%2$s is requesting your username and password. The site says: “%1$s”</string>
[task 2020-05-06T00:14:56.201Z]       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[task 2020-05-06T00:14:56.201Z]       /builds/worker/checkouts/src/components/browser/engine-system/src/main/res/values-nv/strings.xml:7: Conflicting number of arguments here
[task 2020-05-06T00:14:56.201Z]   /builds/worker/checkouts/src/components/browser/engine-system/src/main/res/values/strings.xml:12: Error: Inconsistent number of arguments in formatting string mozac_browser_engine_system_auth_no_realm_message; found both 0 and 1 [StringFormatCount]
[task 2020-05-06T00:14:56.201Z]       <string name="mozac_browser_engine_system_auth_no_realm_message">%1$s is requesting your username and password.</string>
[task 2020-05-06T00:14:56.201Z]       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[task 2020-05-06T00:14:56.202Z]       /builds/worker/checkouts/src/components/browser/engine-system/src/main/res/values-nv/strings.xml:9: Conflicting number of arguments here```

Reply:
cc: @flodolo

Reply:
I've unapproved the string. 

Having said that, I looked at the comment and it makes absolutely no sense, can someone fix it in a follow-up?

Reply:
@Amejia481 ^

Reply:
This is addressed here https://github.com/mozilla-mobile/android-components/pull/7014

Reply:
Sorry for the delay, I missed this comment.